lib/committee/schema_validator/option.rb

```diff
@@ -1,6 +1,13 @@
 class Committee::SchemaValidator
   class Option
-    attr_reader :coerce_recursive, :params_key, :allow_form_params, :allow_query_params, :optimistic_json, :coerce_form_params, :headers_key, :coerce_query_params, :coerce_path_params, :check_content_type, :check_header, :coerce_date_times, :prefix, :validate_success_only
+    attr_reader :coerce_recursive, :params_key,
+                :allow_form_params, :allow_query_params,
+                :optimistic_json,
+                :coerce_form_params, :headers_key,
+                :coerce_query_params, :coerce_path_params,
+                :check_content_type, :check_header,
+                :coerce_date_times, :prefix,
+                :validate_success_only, :allow_get_body
```

Comment:
Since we're deconstructing this anyway, what do you think about one option per line? It makes future changes much cleaner.